src/platform-includes/sourcemaps/upload/cli/javascript.mdx

```diff
@@ -1,52 +1,74 @@
 
-Sentry uses [releases](/product/releases/) to match the correct source maps to your events. To create a new release, run the following command (for example, during publishing):
-
-```bash
-sentry-cli releases new <release_name>
+```javascript
+Sentry.init({
+  // Make sure this value is identical to the name you give the release that you
+  // will create below using Sentry CLI
+  release: "<release_name>",
+});
 ```
 
+## 2. Upload Source Maps Using Sentry CLI
+
 <Note>
 
-The release name must be **unique within your organization** and match the `release` option in your SDK initialization code. Then, use the `upload-sourcemaps` command to scan a folder for source maps, process them, and upload them to Sentry.
+You can find instructions on how to install Sentry CLI here: https://docs.sentry.io/product/cli/installation/
+
+For more info on Sentry CLI configuration visit the [Sentry CLI configuration docs](/product/cli/configuration/).
 
 </Note>
 
-```bash
-sentry-cli releases files <release_name> upload-sourcemaps /path/to/files
-```
+Make sure Sentry CLI is configured for your project. You can use a `.sentryclirc` file or environment variables:
 
 <Note>
 
-You can find the artifacts uploaded to Sentry by navigating to **[Project] > Project Settings > Source Maps**.
+Visit the [auth token user settings page](https://sentry.io/settings/account/api/auth-tokens/) to find your auth tokens.
 
 </Note>
 
-This command will upload all files ending in _.js_ and _.map_ to the specified release. If you wish to change these extensions – for example, to upload typescript sources – use the `--ext` option:
+```ìni {filename:.sentryclirc}
+[auth]
+token=your-auth-token
+
+[defaults]
+org=___ORG_SLUG___
+project=___PROJECT_SLUG___
+```
 
 ```bash
-sentry-cli releases files <release_name> upload-sourcemaps --ext ts --ext map /path/to/files
+export SENTRY_AUTH_TOKEN=your-auth-token
+export SENTRY_ORG=___ORG_SLUG___
+export SENTRY_PROJECT=___PROJECT_SLUG___
 ```
 
-Until now, the release is in a draft state (“_unreleased_”). Once all source maps have been uploaded, and your app has been published successfully, finalize the release with the following command:
+Then, create a release with Sentry CLI. The release name must be unique within your organization:
 
 ```bash
-sentry-cli releases finalize <release_name>
+sentry-cli releases new <release_name>
 ```
 
-For convenience, you can pass the `--finalize` flag to the `new` command, which will immediately finalize the release.
+Next, upload your release artifacts (source files and source maps):
 
-For more information, see our [`sentry-cli` docs](/product/cli/releases/#managing-release-artifacts).
+```bash
+sentry-cli releases files <release_name> upload-sourcemaps /path/to/files
+```
 
-<PlatformSection notSupported={["node"]} >
+<Note>
+
+You can verify your uploaded artifacts by navigating to **[Project] > Project Settings (gear icon in top right corner) > Source Maps** on Sentry.
```

Comment:
```suggestion
You can verify that your upload was successful by navigating to **[Project] > Project Settings (gear icon in top right corner) > Source Maps** on Sentry.
```

FWIW, if we send them via Releases -> specific release -> Source Maps (bottom of RH sidebar), it's 3 clicks rather than 4. Up to you.